puma_motor_driver/src/driver.cpp

```diff
@@ -200,7 +200,7 @@ void Driver::verifyParams()
       }
       break;
     case 7:
-      if (fabs(getP() - gain_p_) < double(1<<16))
+      if (fabs(getP() - gain_p_) < (1/double(1>>16)))
```

Comment:
Ideal would be to verify at the message data level rather than after the float conversion, but I think that ship has sailed at this point. For now, consider a static `floatAlmostEqual` function?